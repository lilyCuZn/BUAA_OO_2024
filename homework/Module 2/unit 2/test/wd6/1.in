[1.3]63-FROM-8-TO-3
[1.6]34-FROM-4-TO-2
[2.2]33-FROM-8-TO-5
[2.8]49-FROM-7-TO-2
[3.0]67-FROM-2-TO-1
[3.0]57-FROM-9-TO-6
[3.1]54-FROM-3-TO-10
[3.9]12-FROM-1-TO-11
[4.2]46-FROM-9-TO-11
[5.6]42-FROM-5-TO-10
[6.2]30-FROM-8-TO-6
[6.8]16-FROM-3-TO-2
[7.0]68-FROM-3-TO-1
[8.0]4-FROM-4-TO-7
[8.3]47-FROM-11-TO-3
[9.3]RESET-Elevator-5-4-0.5
[9.7]39-FROM-10-TO-5
[10.0]41-FROM-5-TO-11
[10.3]7-FROM-2-TO-11
[11.1]29-FROM-1-TO-2
[11.3]8-FROM-7-TO-2
[13.2]1-FROM-2-TO-11
[13.5]2-FROM-7-TO-4
[15.0]60-FROM-5-TO-1
[15.3]48-FROM-11-TO-7
[16.9]18-FROM-11-TO-5
[17.2]36-FROM-1-TO-4
[17.7]25-FROM-10-TO-2
[17.7]64-FROM-7-TO-8
[19.3]5-FROM-8-TO-2
[20.4]52-FROM-10-TO-4
[22.6]45-FROM-6-TO-8
[24.4]17-FROM-6-TO-5
[25.3]RESET-Elevator-2-7-0.5
[25.9]24-FROM-7-TO-1
[27.3]56-FROM-5-TO-3
[27.3]66-FROM-1-TO-3
[28.0]RESET-Elevator-6-6-0.5
[29.1]58-FROM-10-TO-9
[29.6]31-FROM-7-TO-2
[30.0]23-FROM-8-TO-9
[30.2]RESET-Elevator-3-5-0.5
[31.8]20-FROM-5-TO-6
[32.0]19-FROM-6-TO-10
[32.2]62-FROM-9-TO-5
[33.0]28-FROM-2-TO-9
[33.1]RESET-Elevator-6-5-0.5
[33.2]51-FROM-5-TO-10
[33.3]32-FROM-11-TO-3
[33.8]65-FROM-1-TO-8
[34.7]RESET-Elevator-4-6-0.2
[35.3]RESET-Elevator-3-6-0.3
[37.3]37-FROM-4-TO-6
[38.2]RESET-Elevator-6-5-0.2
[38.3]61-FROM-1-TO-6
[38.6]3-FROM-1-TO-10
[38.8]RESET-Elevator-5-5-0.2
[39.8]44-FROM-10-TO-3
[40.7]9-FROM-6-TO-1
[41.0]11-FROM-7-TO-4
[41.5]38-FROM-7-TO-4
[43.3]RESET-Elevator-6-8-0.4
[43.9]RESET-Elevator-5-5-0.5
[44.9]35-FROM-3-TO-8
[46.0]6-FROM-11-TO-8
[46.6]22-FROM-11-TO-9
[47.1]40-FROM-9-TO-7
[47.4]15-FROM-8-TO-9
[49.7]59-FROM-3-TO-9
[50.0]27-FROM-9-TO-4
