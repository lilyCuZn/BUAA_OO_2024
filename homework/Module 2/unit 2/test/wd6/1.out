[   1.4210]RECEIVE-63-1
[   1.7150]RECEIVE-34-2
[   1.8230]ARRIVE-2-1
[   2.1160]ARRIVE-2-2
[   2.2230]ARRIVE-3-1
[   2.3150]RECEIVE-33-3
[   2.5160]ARRIVE-3-2
[   2.6240]ARRIVE-4-1
[   2.7160]ARRIVE-2-3
[   2.9160]RECEIVE-49-2
[   2.9170]ARRIVE-4-2
[   3.0240]ARRIVE-5-1
[   3.1150]RECEIVE-67-4
[   3.1160]ARRIVE-3-3
[   3.1170]RECEIVE-57-5
[   3.2150]RECEIVE-54-6
[   3.3170]ARRIVE-5-2
[   3.4250]ARRIVE-6-1
[   3.5180]ARRIVE-2-4
[   3.5180]ARRIVE-4-3
[   3.5180]ARRIVE-2-5
[   3.5190]OPEN-2-4
[   3.6160]ARRIVE-2-6
[   3.7170]ARRIVE-6-2
[   3.7190]IN-67-2-4
[   3.8250]ARRIVE-7-1
[   3.9180]ARRIVE-5-3
[   3.9190]ARRIVE-3-5
[   3.9190]CLOSE-2-4
[   4.0160]ARRIVE-3-6
[   4.0160]OPEN-3-6
[   4.0170]RECEIVE-12-4
[   4.1180]ARRIVE-7-2
[   4.1180]OPEN-7-2
[   4.2170]IN-54-3-6
[   4.2260]ARRIVE-8-1
[   4.2260]OPEN-8-1
[   4.3160]RECEIVE-46-6
[   4.3180]IN-49-7-2
[   4.3190]ARRIVE-4-5
[   4.3200]ARRIVE-1-4
[   4.3200]OPEN-1-4
[   4.3210]ARRIVE-6-3
[   4.4170]CLOSE-3-6
[   4.4260]IN-63-8-1
[   4.5190]CLOSE-7-2
[   4.5200]OUT-67-1-4
[   4.6260]CLOSE-8-1
[   4.7200]ARRIVE-5-5
[   4.7200]CLOSE-1-4
[   4.7200]OPEN-1-4
[   4.7210]ARRIVE-7-3
[   4.8170]ARRIVE-4-6
[   4.9190]ARRIVE-6-2
[   4.9210]IN-12-1-4
[   5.0270]ARRIVE-7-1
[   5.1200]ARRIVE-6-5
[   5.1210]ARRIVE-8-3
[   5.1210]OPEN-8-3
[   5.1210]CLOSE-1-4
[   5.2180]ARRIVE-5-6
[   5.3190]ARRIVE-5-2
[   5.3210]IN-33-8-3
[   5.4270]ARRIVE-6-1
[   5.5200]ARRIVE-7-5
[   5.5220]ARRIVE-2-4
[   5.5220]CLOSE-8-3
[   5.6180]ARRIVE-6-6
[   5.7160]RECEIVE-42-3
[   5.7200]ARRIVE-4-2
[   5.7200]OPEN-4-2
[   5.8280]ARRIVE-5-1
[   5.9200]IN-34-4-2
[   5.9210]ARRIVE-8-5
[   5.9220]ARRIVE-3-4
[   5.9220]ARRIVE-7-3
[   6.0240]ARRIVE-7-6
[   6.1200]CLOSE-4-2
[   6.2280]ARRIVE-4-1
[   6.3160]RECEIVE-30-5
[   6.3210]ARRIVE-9-5
[   6.3210]OPEN-9-5
[   6.3220]ARRIVE-4-4
[   6.3220]ARRIVE-6-3
[   6.4250]ARRIVE-8-6
[   6.5210]ARRIVE-3-2
[   6.5220]IN-57-9-5
[   6.6290]ARRIVE-3-1
[   6.6290]OPEN-3-1
[   6.7220]CLOSE-9-5
[   6.7220]ARRIVE-5-4
[   6.7230]ARRIVE-5-3
[   6.7230]OPEN-5-3
[   6.8250]ARRIVE-9-6
[   6.8250]OPEN-9-6
[   6.8290]OUT-63-3-1
[   6.9160]RECEIVE-16-1
[   6.9210]ARRIVE-2-2
[   6.9210]OPEN-2-2
[   6.9230]OUT-33-5-3
[   7.0260]IN-46-9-6
[   7.0290]CLOSE-3-1
[   7.0300]OPEN-3-1
[   7.1150]RECEIVE-68-1
[   7.1210]OUT-49-2-2
[   7.1220]OUT-34-2-2
[   7.1220]ARRIVE-8-5
[   7.1220]OPEN-8-5
[   7.1230]ARRIVE-6-4
[   7.1230]CLOSE-5-3
[   7.1230]OPEN-5-3
[   7.2260]CLOSE-9-6
[   7.2300]IN-16-3-1
[   7.2300]IN-68-3-1
[   7.3220]CLOSE-2-2
[   7.3230]IN-30-8-5
[   7.3230]IN-42-5-3
[   7.4310]CLOSE-3-1
[   7.5230]ARRIVE-7-4
[   7.5240]CLOSE-8-5
[   7.5240]CLOSE-5-3
[   7.6260]ARRIVE-10-6
[   7.6270]OPEN-10-6
[   7.8270]OUT-54-10-6
[   7.8310]ARRIVE-2-1
[   7.8310]OPEN-2-1
[   7.9230]ARRIVE-8-4
[   7.9240]ARRIVE-7-5
[   7.9240]ARRIVE-6-3
[   8.0270]CLOSE-10-6
[   8.0320]OUT-16-2-1
[   8.1230]RECEIVE-4-2
[   8.2320]CLOSE-2-1
[   8.3240]ARRIVE-9-4
[   8.3240]ARRIVE-6-5
[   8.3240]OPEN-6-5
[   8.3240]ARRIVE-7-3
[   8.4170]RECEIVE-47-6
[   8.4270]ARRIVE-11-6
[   8.4280]OPEN-11-6
[   8.5240]ARRIVE-3-2
[   8.5240]OUT-57-6-5
[   8.5250]OUT-30-6-5
[   8.6280]OUT-46-11-6
[   8.6330]ARRIVE-1-1
[   8.6330]OPEN-1-1
[   8.7240]ARRIVE-10-4
[   8.7250]ARRIVE-8-3
[   8.7250]CLOSE-6-5
[   8.8280]CLOSE-11-6
[   8.8280]OPEN-11-6
[   8.8340]OUT-68-1-1
[   8.9240]ARRIVE-4-2
[   8.9240]OPEN-4-2
[   9.0290]IN-47-11-6
[   9.0350]CLOSE-1-1
[   9.1240]ARRIVE-11-4
[   9.1240]OPEN-11-4
[   9.1250]IN-4-4-2
[   9.1250]ARRIVE-9-3
[   9.2290]CLOSE-11-6
[   9.3250]OUT-12-11-4
[   9.3250]CLOSE-4-2
[   9.4150]RESET_ACCEPT-5-4-0.5
[   9.4160]RESET_BEGIN-5
[   9.5250]CLOSE-11-4
[   9.5250]ARRIVE-10-3
[   9.5250]OPEN-10-3
[   9.6290]ARRIVE-10-6
[   9.7250]ARRIVE-5-2
[   9.7250]OUT-42-10-3
[   9.8150]RECEIVE-39-3
[   9.9260]CLOSE-10-3
[   9.9260]OPEN-10-3
[  10.0300]ARRIVE-9-6
[  10.1150]RECEIVE-41-2
[  10.1260]ARRIVE-6-2
[  10.1260]IN-39-10-3
[  10.3260]CLOSE-10-3
[  10.4150]RECEIVE-7-1
[  10.4300]ARRIVE-8-6
[  10.5260]ARRIVE-7-2
[  10.5260]OPEN-7-2
[  10.6160]RESET_END-5
[  10.7260]OUT-4-7-2
[  10.7270]ARRIVE-9-3
[  10.8150]ARRIVE-2-1
[  10.8150]OPEN-2-1
[  10.8300]ARRIVE-7-6
[  10.9270]CLOSE-7-2
[  11.0150]IN-7-2-1
[  11.1270]ARRIVE-8-3
[  11.2160]CLOSE-2-1
[  11.2160]RECEIVE-29-5
[  11.2300]ARRIVE-6-6
[  11.3270]ARRIVE-6-2
[  11.4150]RECEIVE-8-3
[  11.5270]ARRIVE-7-3
[  11.5270]OPEN-7-3
[  11.6160]ARRIVE-3-1
[  11.6310]ARRIVE-5-6
[  11.7170]ARRIVE-5-5
[  11.7270]ARRIVE-5-2
[  11.7270]OPEN-5-2
[  11.7280]IN-8-7-3
[  11.9280]CLOSE-7-3
[  11.9300]IN-41-5-2
[  12.0160]ARRIVE-4-1
[  12.0310]ARRIVE-4-6
[  12.1310]CLOSE-5-2
[  12.2170]ARRIVE-4-5
[  12.3280]ARRIVE-6-3
[  12.4170]ARRIVE-5-1
[  12.4310]ARRIVE-3-6
[  12.4320]OPEN-3-6
[  12.5310]ARRIVE-6-2
[  12.6320]OUT-47-3-6
[  12.7170]ARRIVE-3-5
[  12.7290]ARRIVE-5-3
[  12.7290]OPEN-5-3
[  12.8170]ARRIVE-6-1
[  12.8320]CLOSE-3-6
[  12.9290]OUT-39-5-3
[  12.9310]ARRIVE-7-2
[  13.1290]CLOSE-5-3
[  13.2170]ARRIVE-7-1
[  13.2180]ARRIVE-2-5
[  13.3160]RECEIVE-1-6
[  13.3310]ARRIVE-8-2
[  13.5300]ARRIVE-4-3
[  13.6150]RECEIVE-2-4
[  13.6180]ARRIVE-8-1
[  13.7170]ARRIVE-2-6
[  13.7170]OPEN-2-6
[  13.7180]ARRIVE-1-5
[  13.7180]OPEN-1-5
[  13.7320]ARRIVE-9-2
[  13.9170]IN-1-2-6
[  13.9180]IN-29-1-5
[  13.9300]ARRIVE-3-3
[  14.0150]ARRIVE-10-4
[  14.0180]ARRIVE-9-1
[  14.1170]CLOSE-2-6
[  14.1180]CLOSE-1-5
[  14.1320]ARRIVE-10-2
[  14.3350]ARRIVE-2-3
[  14.3350]OPEN-2-3
[  14.4160]ARRIVE-9-4
[  14.4180]ARRIVE-10-1
[  14.5180]ARRIVE-3-6
[  14.5320]ARRIVE-11-2
[  14.5320]OPEN-11-2
[  14.5350]OUT-8-2-3
[  14.6190]ARRIVE-2-5
[  14.6190]OPEN-2-5
[  14.7330]OUT-41-11-2
[  14.7360]CLOSE-2-3
[  14.8160]ARRIVE-8-4
[  14.8180]ARRIVE-11-1
[  14.8180]OPEN-11-1
[  14.8190]OUT-29-2-5
[  14.9180]ARRIVE-4-6
[  14.9330]CLOSE-11-2
[  15.0190]OUT-7-11-1
[  15.0190]CLOSE-2-5
[  15.1160]RECEIVE-60-3
[  15.2160]ARRIVE-7-4
[  15.2170]OPEN-7-4
[  15.2190]CLOSE-11-1
[  15.3180]ARRIVE-5-6
[  15.4150]RECEIVE-48-1
[  15.4150]OPEN-11-1
[  15.4170]IN-2-7-4
[  15.5170]ARRIVE-3-3
[  15.6150]IN-48-11-1
[  15.6170]CLOSE-7-4
[  15.7190]ARRIVE-6-6
[  15.8160]CLOSE-11-1
[  15.9170]ARRIVE-4-3
[  16.0170]ARRIVE-6-4
[  16.1190]ARRIVE-7-6
[  16.2160]ARRIVE-10-1
[  16.3170]ARRIVE-5-3
[  16.3170]OPEN-5-3
[  16.4170]ARRIVE-5-4
[  16.5170]IN-60-5-3
[  16.5190]ARRIVE-8-6
[  16.6160]ARRIVE-9-1
[  16.7180]CLOSE-5-3
[  16.8180]ARRIVE-4-4
[  16.8180]OPEN-4-4
[  16.9190]ARRIVE-9-6
[  17.0160]RECEIVE-18-2
[  17.0160]OPEN-11-2
[  17.0170]ARRIVE-8-1
[  17.0180]OUT-2-4-4
[  17.1180]ARRIVE-4-3
[  17.2160]IN-18-11-2
[  17.2180]CLOSE-4-4
[  17.3150]RECEIVE-36-5
[  17.3200]ARRIVE-10-6
[  17.4170]CLOSE-11-2
[  17.4170]ARRIVE-7-1
[  17.4170]OPEN-7-1
[  17.5180]ARRIVE-3-3
[  17.6170]OUT-48-7-1
[  17.7200]ARRIVE-11-6
[  17.7200]OPEN-11-6
[  17.8160]ARRIVE-1-5
[  17.8160]OPEN-1-5
[  17.8170]ARRIVE-10-2
[  17.8190]CLOSE-7-1
[  17.8650]RECEIVE-25-2
[  17.8870]RECEIVE-64-1
[  17.8870]OPEN-7-1
[  17.9210]OUT-1-11-6
[  17.9210]ARRIVE-2-3
[  18.0160]IN-36-1-5
[  18.0870]IN-64-7-1
[  18.1210]CLOSE-11-6
[  18.2160]CLOSE-1-5
[  18.2170]ARRIVE-9-2
[  18.2880]CLOSE-7-1
[  18.3210]ARRIVE-1-3
[  18.3210]OPEN-1-3
[  18.5220]OUT-60-1-3
[  18.6170]ARRIVE-8-2
[  18.6880]ARRIVE-8-1
[  18.6880]OPEN-8-1
[  18.7170]ARRIVE-2-5
[  18.7220]CLOSE-1-3
[  18.8890]OUT-64-8-1
[  19.0180]ARRIVE-7-2
[  19.0890]CLOSE-8-1
[  19.2170]ARRIVE-3-5
[  19.4160]RECEIVE-5-1
[  19.4160]OPEN-8-1
[  19.4180]ARRIVE-6-2
[  19.6200]IN-5-8-1
[  19.7180]ARRIVE-4-5
[  19.7180]OPEN-4-5
[  19.8180]ARRIVE-5-2
[  19.8180]OPEN-5-2
[  19.8210]CLOSE-8-1
[  19.9180]OUT-36-4-5
[  20.0190]OUT-18-5-2
[  20.1180]CLOSE-4-5
[  20.2190]CLOSE-5-2
[  20.2210]ARRIVE-7-1
[  20.5160]RECEIVE-52-6
[  20.6190]ARRIVE-6-2
[  20.6210]ARRIVE-6-1
[  20.9160]ARRIVE-10-6
[  20.9160]OPEN-10-6
[  21.0190]ARRIVE-7-2
[  21.0210]ARRIVE-5-1
[  21.1160]IN-52-10-6
[  21.3170]CLOSE-10-6
[  21.4200]ARRIVE-8-2
[  21.4210]ARRIVE-4-1
[  21.7170]ARRIVE-9-6
[  21.8200]ARRIVE-9-2
[  21.8210]ARRIVE-3-1
[  22.1170]ARRIVE-8-6
[  22.2200]ARRIVE-10-2
[  22.2200]OPEN-10-2
[  22.2220]ARRIVE-2-1
[  22.2220]OPEN-2-1
[  22.4200]IN-25-10-2
[  22.4220]OUT-5-2-1
[  22.5170]ARRIVE-7-6
[  22.6220]CLOSE-2-1
[  22.6220]CLOSE-10-2
[  22.7190]RECEIVE-45-4
[  22.9180]ARRIVE-6-6
[  23.0220]ARRIVE-9-2
[  23.1190]ARRIVE-5-4
[  23.3180]ARRIVE-5-6
[  23.4230]ARRIVE-8-2
[  23.5190]ARRIVE-6-4
[  23.5190]OPEN-6-4
[  23.7180]ARRIVE-4-6
[  23.7180]OPEN-4-6
[  23.7190]IN-45-6-4
[  23.8230]ARRIVE-7-2
[  23.9190]OUT-52-4-6
[  23.9200]CLOSE-6-4
[  24.1190]CLOSE-4-6
[  24.2230]ARRIVE-6-2
[  24.3200]ARRIVE-7-4
[  24.5160]RECEIVE-17-6
[  24.6230]ARRIVE-5-2
